php_phongo.c

```diff
@@ -2016,6 +2016,18 @@ static void php_phongo_client_dtor(void *client)
 }
 #endif
 
+static void _phongo_check_json (void) __attribute__((constructor));
+static void _phongo_check_json (void)
+{
+	DL_HANDLE handle = DL_LOAD(NULL);
+	zend_class_entry *json = (zend_class_entry *)DL_FETCH_SYMBOL (handle, "php_json_serializable_ce");
```

Comment:
I have doubt about this working if php_json_serializable_ce is not publicly exported.